assets/blocks/reader-registration/edit.js

```diff
@@ -206,15 +206,35 @@ export default function ReaderRegistrationEdit( {
 				{ editedState === 'initial' && (
 					<div className={ `newspack-registration ${ className }` }>
 						<form onSubmit={ ev => ev.preventDefault() }>
-							<RichText
-								onChange={ value => setAttributes( { title: value } ) }
-								placeholder={ __( 'Block title…', 'newspack' ) }
-								value={ title }
-								tagName="h2"
-							/>
+							<div className="newspack-registration__header">
+								<RichText
+									onChange={ value => setAttributes( { title: value } ) }
+									placeholder={ __( 'Write title…', 'newspack' ) }
```

Comment:
Nit: the tone here feels weird, how about 

```suggestion
									placeholder={ __( 'Add title…', 'newspack' ) }
```

like the WP post editor? And then same with the description input.

Reply:
I copied the wording from the Cover block but I guess there are a lot of inconsistencies between blocks....
"Write content...", "Heading", "Add quote", "Content..."

Changed to "Add title" and "Add description" (no elipsis) in https://github.com/Automattic/newspack-plugin/pull/1967/commits/12c1a139669fd392e57d697f92c62cbab3096bfe